### PR TITLE
[MAISTRA-655] Updated Istio images to 1.0.0

### PR DESCRIPTION
**Pull request by brian-avery**

### CODE REVIEW

File: Dockerfile.grafana [hunk 1]
```diff
@@ -2,18 +2,18 @@ FROM registry.access.redhat.com/ubi8/ubi-minimal:latest
...
 LABEL io.k8s.display-name="Red Hat Grafana"
 LABEL io.openshift.tags="istio"
 LABEL io.openshift.expose-services="3000"
 LABEL maintainer="Istio Feedback <istio-feedback@redhat.com>"
 ENV container="oci"
-ENV ISTIO_VERSION=1.1.8
+ENV ISTIO_VERSION=1.1.11
 
-RUN curl -Ls https://copr.fedorainfracloud.org/coprs/g/maistra/grafana/repo/epel-7/group_maistra-grafana-epel-7.repo -o /etc/yum.repos.d/openshift-istio-grafana-epel-7.repo && \
+RUN curl -Ls https://copr.fedorainfracloud.org/coprs/g/maistra/istio/repo/epel-7/group_maistra-grafana-epel-7.repo -o /etc/yum.repos.d/openshift-istio-grafana-epel-7.repo && \
```
Comment (jwendell) [inline Dockerfile.grafana:16]: is this correct?

File: Dockerfile.prometheus [hunk 2]
```diff
@@ -2,18 +2,18 @@ FROM registry.access.redhat.com/ubi8/ubi-minimal:latest
...
 LABEL io.k8s.display-name="Red Hat Prometheus"
 LABEL io.openshift.tags="istio"
 LABEL io.openshift.expose-services="9090"
 LABEL maintainer="Istio Feedback <istio-feedback@redhat.com>"
 ENV container="oci"
-ENV ISTIO_VERSION=1.1.8
+ENV ISTIO_VERSION=1.1.11
 
-RUN curl -Ls https://copr.fedorainfracloud.org/coprs/g/maistra/prometheus/repo/epel-7/group_maistra-prometheus-epel-7.repo -o /etc/yum.repos.d/openshift-istio-prometheus-epel-7.repo && \
+RUN curl -Ls https://copr.fedorainfracloud.org/coprs/g/maistra/istio/repo/epel-7/group_maistra-prometheus-epel-7.repo -o /etc/yum.repos.d/openshift-istio-prometheus-epel-7.repo && \
```
Comment (jwendell) [inline Dockerfile.prometheus:16]: is this correct?

Comment (brian-avery) [reply Dockerfile.prometheus:16]: I've been building Prometheus and Grafana together with the other istio components in the Istio and Istio.el8 repos. I think that makes sense given that we use them for all of the other Istio components as well. What's your thought?

**Comment by dmitri-d** [comment]
I finally understand what you were asking. Keeping grafana and prometheus under istio repository sounds ok to me.

**Review by dmitri-d — APPROVED** [review]
lgtm, with or without moving grafana and prometheus to istio project

**Review by jwendell — APPROVED** [review]

**Comment by jwendell** [comment]
Please put the Jira number in the title/commit message

**Pull request merged**